airflow/jobs/dag_processor_job_runner.py

```diff
@@ -52,7 +56,7 @@ def __init__(
         self.processor = processor
         self.processor.heartbeat = lambda: perform_heartbeat(
             job=self.job,
-            heartbeat_callback=empty_callback,
```

Comment:
We can probably get rid of empty_callback too?

Reply:
Removed it. Thanks!